pkg/manifests/template/helm.go

```diff
@@ -168,7 +168,11 @@ func (h *helm) values(svc *console.GetServiceDeploymentForAgent_ServiceDeploymen
 		}
 	}
 
-	overrides, err := h.valuesFile(svc, "values.yaml.static")
+	filename := "values.yaml.static"
```

Comment:
what's the issue this is trying to protect against?

Reply:
I have `values.yaml` file in my test scenario. I've noticed it was never taken into consideration. Finally my values map was always empty

Reply:
I feel like that's definitely unusual, i'm pretty sure base values.yaml is always there just not in the values overrides

Reply:
I've changed my test `values.yaml` to `values.yaml.static`